src/test/java/uk/gov/hmcts/reform/professionalapi/controller/response/OrganisationPbaResponseTest.java

```diff
@@ -34,16 +29,6 @@ public void testGetOrganisationPbaResponse() throws Exception {
         OrganisationPbaResponse organisationPbaResponse = new OrganisationPbaResponse(organisationMock, true);
 
         OrganisationEntityResponse name;
-
```

Comment:
Please remove this class or wait for other PR where we are removing all the unused test classes

Reply:
We should NOT be removing test classes. I have left that there because the test will need to be added later and therefore it creates more work if we remove the entire class!

Reply:
Ok, I have added an assertion to this class (far preferable to removing all test coverage for it!) Please re-review.